File: Android/BPHC-ID/app/src/main/java/com/srujan/bphc_id/Students.java
package com.srujan.bphc_id;

/**
 * Created by Srujan on 10/14/2015.
 */
public class Students {
        public String Id;
        public String campus_id;
        public String course_id;
        public String name;
        public Students(String cid,String ccid,String name1)
        {
            course_id = ccid;campus_id = cid;name = name1;
        }
}

File: IDExamination/IDSeatingArrangement/IDSeatingArrangement/Students.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IDSeatingArrangement
{
    class Students
    {
        public string Id { get; set; }
        public string Campus_ID { get; set; }
        public string Course_ID { get; set; }
        public string Name { get; set; }
        public Students(string cid,string ccid,string name)
        {
            Course_ID = ccid;Campus_ID = cid;Name = name;
        }
    }
}
